test/langtools/tools/javac/classreader/8226216/T8226216.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
```

Comment:
Tests should not contain the text `Oracle designates ... this code.` or the immediately preceding whitespace.

Reply:
Fixed.